Blackbaud.Addin.TokenAuthentication/SigningKeysCache.cs

```diff
@@ -22,6 +23,7 @@ internal class SigningKeysCache
         /// </summary>
         internal SigningKeysCache()
         {
+            ServicePointManager.SecurityProtocol = SecurityProtocolType.Tls12;
```

Comment:
Any concerns about this affecting the TLS version for the entire application that pulls our library in?

https://github.com/intuit/QuickBooks-V3-DotNET-SDK/issues/54 Quick googling came up with this.

Reply:
They ultimately [decided not](https://github.com/intuit/QuickBooks-V3-DotNET-SDK/commit/441c313fa0f63fc1cd2eeb15d3a77587b91f0bd6#r36743337) to set the security protocol--requiring consumers of the library to enable the TLS version we (OAuth 2 service) supports. Our best option may be to document the TLS requirement... which we should probably point out in our [Authorization docs](https://developer.blackbaud.com/skyapi/docs/authorization).

Reply:
Good catch.  Should have figured this would impact the entire app. I ran my test app and printed out the SecurityProtocol and do in fact see it set to `Tls12`.  So this should be on the app developer.  I'll remove it.